packages/web/src/components/collection/CollectionCard.test.tsx

```diff
@@ -0,0 +1,100 @@
+import { SquareSizes } from '@audius/common/models'
+import { Text } from '@audius/harmony'
+import { Routes, Route } from 'react-router-dom-v5-compat'
+import { describe, it, expect } from 'vitest'
+
+import { render, screen } from 'test/test-utils'
+
+import { CollectionCard } from './CollectionCard'
+
+function renderCollectionCard() {
+  return render(
+    <Routes>
+      <Route path='/' element={<CollectionCard id={1} size='s' />} />
+      <Route
+        path='/test-user/test-collection'
+        element={<Text variant='heading'>Test Collection Page</Text>}
+      />
+      <Route
+        path='/test-user'
+        element={<Text variant='heading'>Test User Page</Text>}
+      />
+    </Routes>,
```

Comment:
woah so dope to see the routes in a test like this

Reply:
yeah! i thought this worked super nicely. It's particularly helpful for the card itself cause it just has a click-handler, not a link, so the only way to ensure it actually works is to do something like this.